new_ftm_language_creation/normalize_audio_filenames.py

```
@@ -0,0 +1,95 @@
+def normalize_audio_filenames(language, root_folder):
+    """Rename audio files to match NFC normalization expected by JSON"""
+    json_path = construct_json_path(root_folder, language)
+    audios_folder = os.path.join(root_folder, language, "audios")
+    
+    if not os.path.exists(audios_folder):
+        print(f"Audios folder does not exist: {audios_folder}")
+        return
+    
+    # Get expected filenames from JSON (already in NFC from JSON parsing)
+    expected_files = find_prompt_audios(json_path)
+    expected_normalized = {normalize_unicode(f) for f in expected_files}
```

Ruff (0.15.0)

[error] Local variable `expected_normalized` is assigned to but never used

Remove assignment to unused variable `expected_normalized`

(F841)